### DISCUSSION TITLE
winlogbeat issues

### DISCUSSION THREAD

**Discussion by DinhDiesel** [Security-Onion-Solutions/securityonion#5808] [Unsupported Versions]
I've downloaded winlogbeat from Security Onion Console (7.13.4) but theres missing files when I try to follow directions from Add Data section of Kibana.  This ".\install-service-winlogbeat.ps1"  is missing, or is 7.15.0(Elastic downloads) compatible with security onion 2.3.8? just wondering how to start the service? Also in the 2.3 Documentation it says basically not to use ElasticSearch and use LogStash which contradict the directions from the kibana section of winlogbeat (add data)...some guidance would be much appreciated!

**Comment by dougburks** [comment]
Please see our Beats documentation:
https://docs.securityonion.net/en/2.3/beats.html

**Reply by DinhDiesel** [reply]
I've done the steps, disabled the dashboard, and elasticsearch output. Also set to logstash, confirmed the Elastic winlogbeat service is running. From manager, confirmed logstash on so-allow is open for the the subnet I want logstash to come from. I've also attempted to load winlogbeat template manually but i can't get a connection even though I tested output and everything is "OK". am I missing something?

**Reply by dougburks** [reply]
Do you have any network firewall that may be blocking this traffic?

**Comment by DinhDiesel** [comment]
![testoutput](https://user-images.githubusercontent.com/87437722/137924270-f78bf5a3-48e3-42a9-948e-41912064f777.JPG)
Here I disabled logstash, enabled elasticsearch, to manually load winlogbeat template (to index) but unable to. I am attempting this because I cant see my logs from winlogbeat agent....
